content/go/concepts/math-functions/terms/hypot/hypot.md

```diff
@@ -0,0 +1,82 @@
+---
+Title: 'Hypot()'
+Description: 'Returns the square root of the sum of two squares.'
+Subjects: 
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags: 
+  - 'Math'
+  - 'Arithmatic'
+  - 'Functions'
+CatalogContent: 
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Hypot()`** function returns the square root of the sum of two squares. In other words, the method returns the result of the following formula:
+
+hypotenuse = $`\sqrt{(x * x) + (y * y)}`$
```

Comment:
The inline formatting isn't supported in docs. The formula should be inside a code block with the latex call ```tex